synapse/lib/cli.py

```diff
@@ -207,13 +208,39 @@ def __init__(self, item, outp=None, **locs):
 
         self.addCmdClass(CmdHelp)
         self.addCmdClass(CmdQuit)
+        self.addCmdClass(CmdLocals)
+
+    def reflectItem(self):
```

Comment:
This is unused in this PR but used in other pending PRs to replace functionality which is currently in ``synapse.lib.cmdr``